PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,108 @@
+<!--
+  PLEASE READ THE FIRST SECTION :-)
+-->
+
+### Is this a bug report?
+
+(write your answer here)
+
+<!--
+  If you answered "Yes":
+  
+    Please note that your issue will be fixed much faster if you are
+    including the exact reproduction steps and a demo.
+    
+    If you're in a hurry or don't feel confident, it's fine to report bugs with
+    less details, but this makes it less likely they'll get fixed soon.
+
+    In either case, please fill as many fields below as you can.
+
+  If you answered "No":
+
+    If this is a question or a discussion, you may delete this template and write in a free form.
+    Note that we only provide help specific to react-static.
+-->
+
+### Environment
+
+<!--
+  Please fill in all the relevant fields by running these commands in terminal.
+-->
+
+1. `node -v`: 
+2. `npm -v`:
+3. `yarn --version` (if you use Yarn):
+
```

Comment:
Lets do this:
1. `react-static -v`: 
2. `node -v`: 
3. `yarn --version` or `npm -v`: